docs/gettingstarted.md

```diff
@@ -93,80 +90,75 @@ more harm than good since it adds unnecessary complexity. Therefore this tutoria
 folders:
 
-| File/Folder       | Purpose                                 |
+| File/Folder       | Content |
 |-------------------|-----------------------------------------|
-| `web/src/main/kotlin` | Contains the source files of all of your Views, Servlets, REST endpoints, async jobs for your application. You'll focus on this folder for the remainder of this guide. |
-| `web/src/main/webapp` | Contains the Vaadin Theme (a SCSS-based theme which gets compiled to CSS). All Themes inherit from the [Valo Theme](https://vaadin.com/valo). JavaScript files, additional CSS files and images are also placed here.  |
-| `web/src/main/resources` | Contains the logger configuration file (`logback.xml`) |
-| `build.gradle`    | This file defines tasks that can be run from the command line. You should add your own tasks by adding code to this file. There is much you can do with Gradle - you can for example use the ssh plugin to deploy the WAR to your production environment. |
-| `README.md`       | This is a brief instruction manual for your application. You should edit this file to tell others what your application does, how to set it up, and so on. |
-| .gitignore        | This file tells git which files (or patterns) it should ignore. See [Github - Ignoring files](https://help.github.com/articles/ignoring-files/) for more info about ignoring files. |
+| `web/src/main/kotlin` | source files of Views, Servlets, REST endpoints, async jobs |
+| `web/src/main/webapp` | Vaadin theme (a SCSS-based theme which gets compiled to CSS), JavaScript files, additional CSS files and images |
+| `web/src/main/resources` | logger configuration file `logback.xml` |
+| `build.gradle`    | Gradle tasks that can be run from the command line |
+| `README.md`       | brief instruction manual for your application |
+| .gitignore        | list of untracked files and folders |
 
-## Hello, Vaadin-on-Kotlin!<a name="4"></a>
+ Most of the work in this tutorial will happen in the
+`web/src/main/kotlin` folder.
 
-To begin with, let's get some text up on screen quickly. To do this, you need to get a web server running.
+## Starting up the Web Server
 
-### Starting up the Web Server
-
-You actually have a functional VoK application already. To see it, you need to start a web server on your development machine.
-You can do this by running the following in the `vok-helloworld-app` directory:
+You now have a functional VoK application. To see it, run the web server on your development machine.
+From the `vok-helloworld-app` directory, run the following command:
 
 ```bash
 $ ./gradlew clean web:appRun
 ```
 
-This will fire up Jetty, an embeddable Java web server. To see your application in action, open a browser window and navigate
+This will fire up Jetty, an embeddable Java web server. Now open a browser and navigate
  to [http://localhost:8080](http://localhost:8080). You should see the Vaadin-on-Kotlin default information page:
 
-![Welcome VoK](images/welcome_vok.png)
```

Comment:
Why do you suggest to remove the  "welcome" screenshot?

Reply:
This is a mistake. sorry.